### PR TITLE
Test fix

### PR DESCRIPTION
**Pull request by bencmbrook**
This PRs into Jimmy's branch

[Extending timeout](https://github.com/transcend-io/conflux/pull/53#issuecomment-623007761) didn't work - there's an issue with the `new Writer` constructor in `write.test.js` when `globalThis.TransformStream` is defined.

Also changes fixtures to generate JSON data rather than JavaScript code (which was the cause of a tough bug).

### CODE REVIEW

File: test/fixture/fixture.test.js [hunk 1]
```diff
@@ -6,10 +6,11 @@
...
+    export default JSON.parse('{}', reviver);
```
Comment (bencmbrook) [inline test/fixture/fixture.test.js:15]: investigate why this didn't rebuild properly

then change `build.js` to write JSON files, not JS. Make fixture.test.js static and import dynamic JSON, and import the `mod` function

File: package.json [hunk 2]
```diff
@@ -1,21 +1,24 @@
...
     "clean": "rimraf build && rimraf coverage",
-    "build": "npm run clean && mkdir -p build && webpack && npm run build:cp-examples && npm run build:markdown",
+    "build": "yarn clean && mkdir -p build && webpack && yarn build:cp-examples && yarn build:markdown",
     "build:cp-examples": "rsync -a --include '*/' --include 'index.html' --exclude '*' example/ build/example",
     "build:markdown": "markdown-toc -i --bullets '—' README.md && sed -i -e 's/—/-/g' README.md && rm -f README.md-e",
+    "####### Run Examples Locally #######": "",
+    "local-serve:examples": "yarn build && live-server build/example",
     "####### Hooks #######": "",
-    "prepare": "cd example && npm i && cd .."
+    "prepare": "cd example && yarn && cd .."
```
Comment (bencmbrook) [inline package.json:21]: need to keep yarn for CI/CD

File: src/write.test.js [hunk 3]
```diff
@@ -4,72 +4,92 @@ import { Writer, Reader } from './index.js';
...
+const folder = { name: 'folder/', directory: true };
 
 test('All in one big test', async (t) => {
-  const { readable, writable } = new Writer()
-  const writer = writable.getWriter()
-  writer.write(helloWorld)
-  writer.write(fileLikeUtf8)
-  writer.write(folder)
-  writer.close()
+  const { readable, writable } = new Writer();
```
Comment (bencmbrook) [inline src/write.test.js:27]: investigate error on the constructor here

File: package.json (hunk 2 above)
Comment (jimmywarting) [reply package.json:21]: hmm, okey...

**Comment by bencmbrook** [comment]
Closing in favor of #66

**Pull request closed without merging**